Code/UnitTests/EditorTest/Project/Project.cpp

```diff
@@ -76,3 +95,38 @@ ezTestAppRun ezEditorTestProject::RunSubTest(ezInt32 iIdentifier, ezUInt32 uiInv
   }
   return ezTestAppRun::Quit;
 }
+
+ezTestAppRun ezEditorTestProject::CreateCppSolution()
+{
+  ezCppSettings cpp;
+  EZ_TEST_BOOL(cpp.Load().Failed());
+
+  EZ_TEST_BOOL(!ezCppProject::ExistsSolution(cpp));
+
+  cpp.m_sPluginName = "TestPlugin";
+#if defined(_MSC_VER)
```

Comment:
Don't we have done ez macro for this ?

Also you don't need the ifdef, you can directly so the number comparison, macros that aren't defined always return the value zero.

Reply:
Replaced it with `#if EZ_ENABLED(EZ_COMPILER_MSVC)`.
I'm keeping the outer ifdef in preparation for linux generator support.